Solutions/Azure Cloud NGFW by Palo Alto Networks/Hunting Queries/CloudNGFW-PotentialBeaconing.yaml

```diff
@@ -0,0 +1,59 @@
+id: 2f8522fc-7807-4f0a-b53d-458296edab8d
+name: Palo Alto - potential beaconing detected
+description: |
+  'Identifies beaconing patterns from PAN traffic logs based on recurrent timedelta patterns.
+   Reference Blog:https://techcommunity.microsoft.com/t5/microsoft-sentinel-blog/detect-network-beaconing-via-intra-request-time-delta-patterns/ba-p/779586'
+severity: Low
+status: Available
+requiredDataConnectors:
+  - connectorId: CloudNgfwByPAN
+    dataTypes:
+      - fluentbit_CL
+queryFrequency: 1d
+queryPeriod: 2d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1071
+  - T1571
+query: |

```

Comment:
Hi @davidokeyode , The argument value must be summable: a number, timespan or datetime.

Reply:
@v-sabiraj - Not understanding this comment. Are you referring to the use of the **_arg_max_** function. If it is, it seems to be properly used from my review. Any exact issues that you see with it along with guiding documentations?